po/fr.po

```diff
@@ -1893,50 +1897,51 @@ msgstr "nom de l'attribut"
 
 #: rust/libnewsboat/src/filterparser.rs:64
 msgid "one of: =~, ==, =, !~, !=, <=, >=, <, >, between, #, !#"
-msgstr ""
+msgstr "L'un de: =~, ==, =, !~, !=, <=, >=, <, >, between, #, !#"
 
 #: rust/libnewsboat/src/filterparser.rs:65
 msgid "one of: quoted string, range, number"
-msgstr ""
+msgstr "L'un de: quoted string, range, number"
 
 #. The "%{}" thing is a number, a zero-based offset into a string.
 #: rust/libnewsboat/src/filterparser.rs:356
 msgid "Parse error: trailing characters after position %{}: %s"
-msgstr ""
+msgstr "Erreur d'analyse : caractères surnuméraires après la position %{}: %s"
 
 #. The "%{}" thing is a number, a zero-based offset into a string.
 #: rust/libnewsboat/src/filterparser.rs:363
 msgid "Parse error at position %{}: expected %s"
-msgstr ""
+msgstr "Erreur d'analyse à la position%{}: %s était attentu"
```

Comment:
There should be a space before the `%{}` (which will get replaced by a number), and also before the colon.